src/pages/ViewFreaks/ViewFreaks.jsx

```diff
@@ -35,13 +36,15 @@ function ViewFreaks() {
           modalId={ modals.SKILLS }
           isOpen={ openModal === modals.SKILLS }
           setOpenModal={ setOpenModal }
+          filterId="Skills"
         />
         <QueryFilterModal
           title="Projects"
           keywords={ projects }
           modalId={ modals.PROJECTS }
           isOpen={ openModal === modals.PROJECTS }
           setOpenModal={ setOpenModal }
+          filterId="Projects"
```

Comment:
If referencing the skill type, the enum should be used

```suggestion
          filterId={FilterType.projects}
```